android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/DotsIndicator.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun DotsIndicator(
+    totalDots: Int,
+    selectedIndex: Int
+) {
+    LazyRow(
+        modifier = Modifier
+            .wrapContentWidth()
+            .wrapContentHeight()
+    ) {
+        items(totalDots) { index ->
+            if (index == selectedIndex) {
+                Box(
+                    modifier = Modifier
+                        .size(8.dp)
+                        .clip(CircleShape)
+                        .background(Color.White)
+                )
+            } else {
+                Box(
+                    modifier = Modifier
+                        .size(8.dp)
+                        .clip(CircleShape)
+                        .background(Color.White.copy(alpha = 0.2f))
+                )
+            }
+            if (index != totalDots - 1) {
+                Spacer(modifier = Modifier.padding(horizontal = 5.dp))
+            }
```

Comment:
Could you please double-check the padding logic here? I think it would affect to the first indicator.

Reply:
Checked, and it's the right logic. However, I updated to use `Modifier.width` instead to clarify it - https://github.com/suho/kmm-ic/pull/106/commits/292662b6eebc6d1b2d93c12abc3ad4e6f6ecc4b7